client/app/packs/rsc-client-components.js

```
@@ -0,0 +1,11 @@
+'use client';
+
+// RSC client components registration.
+// Components with 'use client' that are used in server components must be
+// available in a client bundle chunk so the React flight client can load them.
+
+import TogglePanel from '../bundles/server-components/components/TogglePanel';
+import ReactOnRails from 'react-on-rails-pro';
```

ESLint

[error] `react-on-rails-pro` import should occur before import of `../bundles/server-components/components/TogglePanel`

(import/order)